### ISSUE TITLE
Check user's permissions

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/conda-forge-webservices#113]
Thinking about whether we should be checking user's permissions before running certain commands. For example, should one be able to ask for a re-rendering to a PR that they don't have push permissions to otherwise (i.e. a feedstock they cannot make changes to)? Any thoughts on this @isuruf?

**Comment by isuruf** [comment]
This should be easy to fix. Although, letting anybody do this has advantages too. This way other people can help with the maintenance, but only a maintainer can merge them. (For eg: I can start the linter on a PR, otherwise only the maintainer, PR author, you and @ocefpaf, will have permissions)

**Comment by ocefpaf** [comment]
> Although, letting anybody do this has advantages too. This way other people can help with the maintenance, but only a maintainer can merge them. 

:+1:

**Comment by jakirkham** [comment]
Indeed. Sure the linter seems pretty harmless. Am more referring to situations where the PR's content changes. Doesn't seem like someone who can't already change that content should be able to through a bot.
